core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -10,18 +10,22 @@ import scalacss.ScalaCssReact._
 
 object ReactTable {
 
-  type Model = Map[String, Any]
+  type ColumnKey = String
+  type Model = Map[ColumnKey, Any]
+  type CellRenderer = (Model, ColumnKey) ⇒ ReactElement
 
-  /**
-   *  ._1: String = column name
-   *  ._2: Option[Any => ReactElement] = custom cell
-   *  ._3: Option[(Model,Model) => Boolean] = sorting function
-   *  ._4: Option[Double] = column width interms of flex property
-   */
-  type Config = (String, Option[Any => ReactElement], Option[(Model, Model) => Boolean],Option[Double])
+  case class ColumnConfig(name: String,
```

Comment:
Fair enough, i just thought it looked a bit clumsy in the demos, but we can leave it like it is